cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -109,7 +109,12 @@
     </phase>
   </test>
 
-
+  <test name="PEM_D_Ld15.5x5_amazon.I2000Clm50FatesRs.cheyenne_gnu.clm-FatesColdSeedDisp">
+    <phase name="COMPARE_base_modpes">
```

Comment:
For some reason this isn't showing up as an expected failure during fates suite regression testing.  @billsacks originally found a discrepancy between the testlist definition and this, but I've since fixed that without seeing the "expected failure" appended to the `./cs.status` output.

Reply:
This worked with the latest retest of fates.